src/main/java/by/savich/project/springboot/service/IphoneService.java

```diff
@@ -35,10 +34,8 @@ public void saveIphone(Iphone iphone) {
     }
 
     public Iphone getIphoneById(int id) {
-        if (iphoneRepository.findById(id) == null) {
-            throw new RuntimeException("No iphone with id " + id + " was found!");
-        }
-        return iphoneRepository.findById(id).get();
+        return iphoneRepository.findById(id).orElseThrow(() ->
```

Comment:
Think about troubleshooting. Probably exception will be logged. It would be useful to have an `id` in the logs.
```java
"IPhone not found id=" + id
```
Use `id=123` for logs. It can be parsed by Splunk etc.
Is it an `IPhone` or secret?